llvm/include/llvm/ADT/GenericUniformityImpl.h

```diff
@@ -396,6 +399,8 @@ template <typename ContextT> class GenericUniformityAnalysisImpl {
 
   void print(raw_ostream &out) const;
 
+  iterator_range<const UseOutsideCycleInfoT *> uses_outside_cycle() const;
```

Comment:
Is it correct that this only tracks uses outside a cycle with divergent exits? This should be in the name or at least in a comment.